docs/content/how-to/kubevirt/create-kubevirt-cluster.md

```diff
@@ -4,228 +4,226 @@ Install a nested OCP cluster running on VMs within a management OCP cluster
-the cluster name `mgmt-cluster` then the management cluster's wildcard DNS
-record will look like this, `*.apps.mgmt-cluster.example.com`. Now if we use
-the management cluster's application DNS entry `apps.mgmt-cluster.example.com`
-as the base domain for a Hypershift KubeVirt cluster called `kv-cluster` then
-the resulting application DNS for the Hypershift cluster becomes
- `*.apps.kv-cluster.apps.mgmt-cluster.example.com` which is a subdomain of the
-management cluster's DNS entry.
-
-**Step 1**
-Enable wildcard routes on the management cluster.
-
-`oc patch ingresscontroller -n openshift-ingress-operator default --type=json -p '[{ "op": "add", "path": "/spec/routeAdmission", "value": {wildcardPolicy: "WildcardsAllowed"}}]'`
-
-**Step 2**
-Create Hypershift KubeVirt cluster with the base domain `apps.<mgmt cluster>.<mgmt cluster base domain>`
-
-If the management cluster is called `mgmt-cluster` and it has the base domain
-`example.com` then the Hypershift base domain should be
-`apps.mgmt-cluster.example.com`. This would be the `--base-domain` cli arg
-passed into the hypershift binary during cluster creation.
-
-**Step 3**
-
-Create an OCP route and ClusterIP service that routes application
-traffic through the management cluster into the Hypershift KubeVirt VMs.
+## Delete a HostedCluster
 
-First detect the Hypershift KubeVirt cluster's ingress NodePort. This requires
-using the kubeconfig for the new Hypershift KubeVirt cluster to retrieve
-the application NodePort service in order to determine what port to route
-ingress traffic to.
+To delete a HostedCluster:
 
-```
-hypershift create kubeconfig > test-kubeconfig
-export HTTPS_NODEPORT=$(oc --kubeconfig test-kubeconfig get services -n openshift-ingress router-nodeport-default -o wide | awk '{print $5}' | awk -F "443:" '{print $2}' | awk -F "/" '{print $1}' | tr -d '[:space:]')
+```shell
+hypershift destroy cluster kubevirt --name $CLUSTER_NAME
 ```
 
-Alter these environment variables to fit your environment.
-```
-export KUBEVIRT_CLUSTER_NAME=kv-cluster
-export KUBEVIRT_CLUSTER_NAMESPACE=clusters-kv-cluster
-export KUBEVIRT_CLUSTER_BASE_DOMAIN=${KUBEVIRT_CLUSTER_NAME}.apps.mgmt-cluster.example.com
-```
+## Optional MetalLB Configuration Steps
 
-Post an OCP Route and Service on the management cluster which routes management
-cluster ingress traffic to the Hypershift KubeVirt cluster ingress service's
-port on the KubeVirt VMs
+A load balancer is required. If MetalLB is in use, here are some example steps
+outlining how to configure MetalLB after [installing MetalLB](https://docs.openshift.com/container-platform/4.12/networking/metallb/metallb-operator-install.html).
 
+**Step 1. Create a MetalLB instance**
+```shell
+oc create -f - <<EOF
+apiVersion: metallb.io/v1beta1
+kind: MetalLB
+metadata:
+  name: metallb
+  namespace: metallb-system
+EOF
 ```
-cat << EOF > apps-ingress-service.yaml
-apiVersion: v1
-kind: Service
```

Comment:
Oh this part will be covered by this [PR](https://github.com/openshift/hypershift/pull/1787) ?  As long as user are building the binary from main branch, we could use the Default Ingress Passthrough feature, right?

Reply:
yes. This also happens to work successfully in MCE 2.2 with any ocp guest cluster that is 4.12.2 or greater. Hypershift got a backport to 4.12 that included this basedoman passthrough work.